js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogInsightsLineChart.tsx

```diff
@@ -195,6 +206,54 @@ export const AssetCatalogInsightsLineChart = React.memo(
       }),
       [renderTooltipFn, rgbColors],
     );
+
+    const chartRef = useRef(null);
+    const onClick = (event: React.MouseEvent<HTMLCanvasElement>) => {
+      if (!chartRef.current) {
+        return;
+      }
+
+      const chart = ChartJS.getChart(chartRef.current);
+      if (!chart) {
+        return;
+      }
+
+      const clickedElements = chart.getElementsAtEventForMode(
+        event as unknown as Event,
```

Comment:
If what you need is the underlying native event, you can try passing `event.nativeEvent` here. That should be a native `MouseEvent`, which extends `Event`, and should allow you to avoid having to cast the value.

Reply:
nice! that worked